src/address.c

```diff
@@ -314,3 +314,80 @@ int verifyP2pkhAddress(char* p2pkh_pubkey, uint8_t len)
     free(dec);
     return true;
 }
+
+/**
+ * @brief This function generates a derived child key from a masterkey using
+ * a custom derived path in string format.
+ * 
+ * @param masterkey The master key from which children are derived from.
+ * @param derived_path The path to derive an address from according to BIP-44.
+ * e.g. m/44h/3h/1/1/1 representing m/44h/3h/account/ischange/index
```

Comment:
only thing i want to add here is bip44 is i believe generally m/44h/3h/1h/1/1 vs ...1/1/1 i think.. want to double check this before ack

Reply:
I've amended the note to `m/44/3` and will push shortly. Are you saying you'd like to include the ' though?

Reply:
' or h doesn't matter, just want it to indicate hardening levels to bip-44 standards since m/44h/3h/1/1/1 isn't a thing, m/44h/3h/1h/1/1 is..

Reply:
Amended again since this but pertinent line for note is L330 and reflected in the code on L386 in getDerivedHDAddress function.